### PR TITLE
an explainer for the demo

### LINKED ISSUE #4062: Issue with "flex-basis": (short summary here please)
MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/flex-basis

#### What information was incorrect, unhelpful, or incomplete?
In the first **"CSS Demo: flex-basis"** it is unclear to which elements flex-basis is applied. Only the property was displayed, not the whole rule. I assumed first that the rule is applied to all items (item-one, item-two, item-three), and was confused why `flex-basis: 200px` made only first item grow 200px wide, and other element to shrink.

#### Specific section or headline?
CSS Demo: flex-basis

#### What did you expect to see?
instead of ` flex-basis: 200px;` 
`.item-one { flex-basis: 200px }`

#### Did you test this? If so, how?

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/css/flex-basis`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/flex-basis
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/flex-basis/index.html
* Last commit: https://github.com/mdn/content/commit/f4fb4c2d90a134366c280eec6cb599825981ad5d
* Document last modified: 2021-03-15T17:55:56.000Z

</details>

### PR DESCRIPTION
**Pull request by rachelandrew**
Fixes #4062 

Adds an explanation of what happens in the Interactive Example, as it isn't very clear and will be hard to make so due to the interaction between the various flex properties.

### CODE REVIEW

File: files/en-us/web/css/flex-basis/index.html [hunk 1]
```diff
@@ -14,7 +14,17 @@
 
 <div>{{EmbedInteractiveExample("pages/css/flex-basis.html")}}</div>
 
-<div class="note">
+<p>In this example the {{cssxref("flex-grow")}} and {{cssxref("flex-shrink")}} properties are both set to <code>1</code> on all three items, indicating that the flex item can grow and shrink from the initial <code>flex-basis</code>.</p>
+
+<p>The demo then changes the <code>flex-basis</code> on the first item. It will then grow and shrink from that flex-basis. This means that, for example, when the <code>flex-basis</code> of the first item is <code>200px</code>, it will start out at 200px but then shrink to fit the space available with the other items being at least <code>min-content</code> sized.</p>
+
+<figure style="padding-bottom: 2em;">
```
Comment (ericwbailey) [inline files/en-us/web/css/flex-basis/index.html:21]: What's our policy on inline styling like this? I assume it's to flag an issue if we need better defaults?

File: files/en-us/web/css/flex-basis/index.html [hunk 2]
```diff
@@ -14,7 +14,17 @@
...
+  <img src="firefox-flex-basis.jpg" alt="The Firefox Flexbox Inspector showing the size of the item once it has shrunk">
```
Comment (ericwbailey) [inline files/en-us/web/css/flex-basis/index.html:22]:
```suggestion
  <img src="firefox-flex-basis.jpg" alt="The Firefox Flexbox Inspector showing the size of the item once it has shrunk.">
```

Periods supply a nice pause for AT to help communicate the end of the alt description.

**Review by ericwbailey — CHANGES_REQUESTED** [review]

File: files/en-us/web/css/flex-basis/index.html (hunk 1 above)
Comment (rachelandrew) [reply files/en-us/web/css/flex-basis/index.html:21]: I'm going to raise it as an issue, if you remove it in DevTools you'll see the problem I'm fixing. I figured I'd do that for now then raise an issue for the site CSS.

Comment (ericwbailey) [reply files/en-us/web/css/flex-basis/index.html:21]: Ah yep, yikes.

**Review by ericwbailey — APPROVED** [review]

**Pull request merged**